cmd/protoc-gen-gorums/gengorums/gorums_func_map.go

```diff
@@ -225,3 +232,24 @@ func mustExecute(t *template.Template, data interface{}) string {
 	}
 	return b.String()
 }
+
+func numStrictOrderingMethods(services []*protogen.Service) int {
```

Comment:
Nitpick: maybe better to use `hasStrictOrderingMethods`, which could return a bool.